templates/aws-standalone-cp/templates/apps/cert-manager.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: HelmRepository
+metadata:
+  name: cert-manager
+  labels:
+    hmc.mirantis.com/managed: "true"
+spec:
+  interval: 24h
+  url: https://charts.jetstack.io
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: {{ include "cluster.name" . }}-cert-manager
+  labels:
+    hmc.mirantis.com/managed: "true"
+spec:
+  chart:
+    metadata:
+      labels:
+        hmc.mirantis.com/managed: "true"
+    spec:
+      chart: cert-manager
+      version: ">=v1.12.3"
```

Comment:
@Schnitzel @wahabmk do we want automatic upgrades here?

Reply:
Yeah good point. I should remove the `>=` and just pin it to a specific version.

Reply:
yea agree, we should not have auto updates, let's pin it.